1-3/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8" >
+  <meta name="description" content="プログラミング学習">
+  <title>javascript課題1-1</title>
+</head>
+<body>
+
+<script>
+
+const task = ['掃除','買い物','散歩'] ;
```

Comment:
`,`の後ろにスペースをあけるのが通例ですので覚えておきましょう。
```suggestion
const task = ['掃除', '買い物', '散歩'] ;
```